theano/compile/tests/test_function_module.py

```diff
@@ -329,35 +342,37 @@ def test_swap_SharedVariable(self):
             for key in cpy.fn.storage_map:
                 if key.name in names:
                     assert map_SV[key.name].container.storage[0] ==\
-                           cpy.fn.storage_map[key][0]
+                        cpy.fn.storage_map[key][0]
 
             second_time = True
 
-    def test_swap_SharedVaraile_with_given(self):
+    def test_swap_SharedVariable_with_given(self):
         """
         A special testcase for logistic_sgd.py in Deep Learning Tutorial
         This test assert that SharedVariable in different function have same storage
         """
-        train_x = theano.shared(value=numpy.random.rand(10,10).astype(config.floatX))
-        test_x = theano.shared(value=numpy.random.rand(10,10).astype(config.floatX))
+        # train_x = theano.shared(value=numpy.random.rand(10, 10).astype(config.floatX))
+        test_x = theano.shared(value=numpy.random.rand(10, 10).astype(config.floatX))
 
-        train_y = theano.shared(value=numpy.random.rand(10,1).astype(config.floatX))
-        test_y = theano.shared(value=numpy.random.rand(10,1).astype(config.floatX))
+        # train_y = theano.shared(value=numpy.random.rand(10, 1).astype(config.floatX))
+        test_y = theano.shared(value=numpy.random.rand(10, 1).astype(config.floatX))
 
         i = T.iscalar('index')
         x = T.vector('x')
         y = T.vector('y')
         # this formular has no sense but for a test
         out = (T.sum(x) - y) ** 2
-        train = theano.function([i], out,
-                                givens={x:train_x[i], y:train_y[i]},
-                                updates={train_x:train_x+0.1})
+        out2 = (T.sum(x) + y) ** 2
+        # train = theano.function([i], out,
+        #                        givens={x: train_x[i], y: train_y[i]},
+        #                        updates={train_x: train_x + 0.1})
 
-        test_def = theano.function([i], out, givens={x:test_x[i], y:test_y[i]})
-        test_cpy = train.copy(swap={train_x:test_x, train_y:test_y},
-                              delete_updates=True)
+        test_def = theano.function([i], out, givens={x: test_x[i], y: test_y[i]})
+        test_def2 = theano.function([i], out2, givens={x: test_x[i], y: test_y[i]})
+        # test_cpy = train.copy(swap={train_x: test_x, train_y: test_y},
+        #                      delete_updates=True)
```

Comment:
Instead of commenting this, can you just not assign the result.  Because otherwise, this disables the test of the copy